cpp/ql/test/library-tests/ir/ir/aliased_ir.expected

```diff
@@ -747,6 +747,42 @@ coroutines.cpp:
 #   87|     m87_4(unknown)             = Chi                                       : total:m87_2, partial:m87_3
 #   87|     r87_5(glval<promise_type>) = VariableAddress[(unnamed local variable)] : 
 #   87|     m87_6(promise_type)        = Uninitialized[(unnamed local variable)]   : &:r87_5
+#   87|     r87_7(glval<promise_type>) = VariableAddress[(unnamed local variable)] : 
+#   87|     r87_8(glval<unknown>)      = FunctionAddress[initial_suspend]          : 
+#   87|     r87_9(suspend_always)      = Call[initial_suspend]                     : func:r87_8, this:r87_7
+#   87|     m87_10(unknown)            = ^CallSideEffect                           : ~m87_4
+#   87|     m87_11(unknown)            = Chi                                       : total:m87_4, partial:m87_10
+#   87|     v87_12(void)               = ^IndirectReadSideEffect[-1]               : &:r87_7, m87_6
+#   87|     m87_13(promise_type)       = ^IndirectMayWriteSideEffect[-1]           : &:r87_7
+#   87|     m87_14(promise_type)       = Chi                                       : total:m87_6, partial:m87_13
+#-----|     v0_1(void)                 = CopyValue                                 : r87_9
+#-----|     r0_2(glval<promise_type>)  = VariableAddress[(unnamed local variable)] : 
+#-----|     r0_3(glval<unknown>)       = FunctionAddress[return_void]              : 
+#-----|     v0_4(void)                 = Call[return_void]                         : func:r0_3, this:r0_2
+#-----|     m0_5(unknown)              = ^CallSideEffect                           : ~m87_11
+#-----|     m0_6(unknown)              = Chi                                       : total:m87_11, partial:m0_5
+#-----|     v0_7(void)                 = ^IndirectReadSideEffect[-1]               : &:r0_2, m87_14
+#-----|     m0_8(promise_type)         = ^IndirectMayWriteSideEffect[-1]           : &:r0_2
+#-----|     m0_9(promise_type)         = Chi                                       : total:m87_14, partial:m0_8
+#   88|     v88_1(void)                = NoOp                                      : 
+#-----|     v0_10(void)                = NoOp                                      : 
+#-----|   Goto (back edge) -> Block 1
```

Comment:
I'm somewhat confused by this. Why are there two blocks here and why is this Goto marked as a back edge?

Reply:
Oh, I didn't spot this either. I'm also confused by this. Will investigate!

Reply:
Aha! I've found the problem:
```ql
private predicate foo(UnaryOperation uo) {
  not exists(uo.getOperand())
}
```
has 4 results in the `co_return_void` function 🤔 So I think there's some extractor issue going on here.

Reply:
I've created an issue for this now.